codelists/api.py

```diff
@@ -48,15 +48,22 @@ def codelists_get(request, owner):
     ]
     """
 
+    filter_kwargs = {}
+
+    if "coding_system_id" in request.GET:
+        filter_kwargs["coding_system_id"] = request.GET["coding_system_id"]
+
     records = []
 
     for cl in sorted(
-        owner.codelists.prefetch_related("handles", "versions"), key=lambda cl: cl.slug
+        owner.codelists.filter(**filter_kwargs).prefetch_related("handles", "versions"),
+        key=lambda cl: cl.slug,
     ):
         record = {
             "full_slug": cl.full_slug(),
             "slug": cl.slug,
             "name": cl.name,
+            "coding_system_id": cl.coding_system_id,
```

Comment:
Really minor thing, but the comment at the start of this function could be updated with the new parameter details.

Reply:
Good point.  Did a quick fix without engaging brain.